Docs/user_guide/about.md

```diff
@@ -22,6 +22,24 @@ The 3D Slicer software is distributed under a BSD-style open source license that
 
 To use Slicer, please read the [3D Slicer Software License Agreement](https://github.com/Slicer/Slicer/blob/master/License.txt) before downloading any binary releases of the Slicer.
 
+### Licensing Context
```

Comment:
"Licensing context" sounds like some additional licensing terms or conditions, which could make things sound more complicated. I would make the section title less official-sounding, something like "Why a custom open-source license?" or something that makes it clear that this is some additional background information ("Rationale", "Motivation", "Background", ...).